### PR TITLE
hide new question links and touch up styling

### PR DESCRIPTION
**Pull request by WKocur**
Hide links to add new questions
Add padding to question tables
Add page containers to question edit/new/show actions

### CODE REVIEW

File: app/views/demographic_questions/show.html.erb [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
-  <h1><%= locale %></h1>
-  <% I18n.locale = locale %>
-  <p>
-    <strong>Text:</strong>
-    <%= @demographic_question.text %>
-  </p>
-<% end %>
+  <% I18n.available_locales.each do |locale| %>
+    <h1><%= locale %></h1>
+    <% I18n.locale = locale %>
```
Comment (drewpotter) [inline app/views/demographic_questions/show.html.erb]: @WKocur do we want to use `Globalize.with_locale(locale)` here like elsewhere?

File: app/views/questions/show.html.erb [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-<% I18n.available_locales.each do |locale| %>
-  <h1><%= locale %></h1>
-  <% I18n.locale = locale %>
-  <p>
-    <strong>Text:</strong>
-    <%= @question.text %>
-  </p>
-<% end %>
+  <% I18n.available_locales.each do |locale| %>
+    <h1><%= locale %></h1>
```
Comment (drewpotter) [inline app/views/questions/show.html.erb]: Same again, do we want to use `Globalize.with_locale(locale)` here?

Comment (WKocur) [reply app/views/questions/show.html.erb]: This is in develop already I think :)

**Comment by WKocur** [comment]
I've resolved the conflicts so the globalize stuff is now in there

**Comment by drewpotter** [comment]
Thanks! I will merge now.

**Pull request merged**